bot/cogs/coding/coding.py

```diff
@@ -93,12 +92,19 @@ async def update_languages(self) -> None:
         """,
         brief="Execute code in a given programming language",
     )
-    async def run(self, ctx: Context, language: str, *, code: str = "") -> t.Union[None, str]:
+    async def run(
+        self, ctx: Context, language: str, *, code: str = ""
+    ) -> t.Union[None, str]:
         """Execute code in a given programming language."""
 
-        options = {"--stats": False, "--wrapped": False}  # the flags to be used when the compiler is needed
+        options = {
+            "--stats": False,
+            "--wrapped": False,
+        }  # the flags to be used when the compiler is needed
 
-        lang = language.strip("`").lower()  # strip the "`" characters to obtain code
+        lang = language.strip(
+            "`"
+        ).lower()  # strip the "`" characters to obtain code
```

Comment:
```suggestion
        # strip the "`" characters to obtain code
        lang = language.strip( "`").lower() 
```
This shouldn't be split.